api/odl2.py

```diff
@@ -0,0 +1,260 @@
+import json
+import logging
+
+from contextlib2 import contextmanager
+from flask_babel import lazy_gettext as _
+from webpub_manifest_parser.opds2.registry import OPDS2LinkRelationsRegistry
+
+from api.odl import ODLAPI
+from core.metadata_layer import FormatData, LicenseData
+from core.model import DeliveryMechanism, Edition, MediaTypes, RightsStatus
+from core.model.configuration import (
+    ConfigurationAttributeType,
+    ConfigurationFactory,
+    ConfigurationGrouping,
+    ConfigurationMetadata,
+    ConfigurationStorage,
+    HasExternalIntegration,
+)
+from core.opds2_import import OPDS2Importer, OPDS2ImportMonitor
+from core.util import first_or_default
+
+
+class ODL2APIConfiguration(ConfigurationGrouping):
+    skipped_license_formats = ConfigurationMetadata(
+        key="odl2_skipped_license_formats",
+        label=_("License formats"),
+        description=_("Name of the data source associated with this collection."),
+        type=ConfigurationAttributeType.LIST,
+        required=False,
+        default=["text/html"],
+    )
+
+
+class ODL2API(ODLAPI):
+    NAME = "ODL 2.0"
+    SETTINGS = ODLAPI.SETTINGS + ODL2APIConfiguration.to_settings()
+
+
+class ODL2Importer(OPDS2Importer, HasExternalIntegration):
+    """Import information and formats from an ODL feed.
+
+    The only change from OPDSImporter is that this importer extracts
+    format information from 'odl:license' tags.
+    """
```

Comment:
Minor, since the base class is right there, but should probably change `OPDSImporter` to `OPDS2Importer` in the docstring.

Reply:
Fixed it, thank you for noticing it!

Reply:
Would you double check this. I'm not sure it says what you want it to say.

Reply:
@tdilauro, @jonathangreen I update the docstring.